crates/matrix-sdk/src/room/timeline/event_item.rs

```diff
@@ -331,10 +331,7 @@ impl Message {
 
     /// Flag indicating this message can be edited by current user.
     pub fn is_editable(&self) -> bool {
-        match self.msgtype() {
-            MessageType::Text(_) => true,
-            _ => false,
-        }
+        matches!(self.msgtype(), MessageType::Text(_))
```

Comment:
Shouldn't this include `self.is_own()`?

Reply:
It's used [here](https://github.com/matrix-org/matrix-rust-sdk/blob/42deb2c9b20ecc0ae24671340ec16c81c10cf234/crates/matrix-sdk/src/room/timeline/event_item.rs#L171), this is inner type.

Reply:
Can we inline the `matches!` there? At least with the current doc string, this method doesn't really makes sense to me.

Reply:
Both inlined with `matches!` now and updated the doc. Does it make sense now?

Reply:
No, that is not what I was thinking. I meant if we only need this function as part of the other one, we should inline this function and remove it.

Reply:
Ok, updated.